libs/partners/ai21/tests/unit_tests/test_llms.py

```diff
@@ -0,0 +1,90 @@
+"""Test AI21 Chat API wrapper."""
+from unittest.mock import call
+
+import pytest
+from ai21.models import (
+    Penalty,
+)
+
+from langchain_ai21 import AI21
+from libs.partners.ai21.tests.unit_tests.conftest import BASIC_EXAMPLE_LLM_PARAMETERS
+
+
+@pytest.mark.requires("ai21")
```

Comment:
I would add a test that raises an exception when there's no API key provided

Reply:
Done for each model